pkg/virt-controller/watch/replicaset_test.go

```diff
@@ -306,12 +307,13 @@ var _ = Describe("Replicaset", func() {
 			vmiFeeder.Modify(modifiedVMI)
 
 			// Expect the re-crate of the VirtualMachineInstance
+			rsInterface.EXPECT().Update(rsCopy).Times(1)
```

Comment:
I see no new unit test which explicitly addresses the cleanup. Can you add one or two?

Reply:
will add it